simplesamlphp/config/default.local.authsources.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Include any necessary changes to the authsources config here.
+ */
+
+// This file should be copied into vendor/simplesamlphp/simplesamlphp/config.
+$root = dirname(__FILE__, 5);
+
+$config['default-sp']['entityID'] = 'https://mrc.stanford.edu';
```

Comment:
Oh, I see what you did here. Totally cheating but it works. I did something similar on sites-pro.stanford.edu

Reply:
:grin: